homework/week-0/sarahkim/AdventureGame/src/nyc/c4q/hyunj0/Air.java

```diff
@@ -0,0 +1,22 @@
+package nyc.c4q.hyunj0;
+/**
+ * Created by c4q-sarahkim on 5/2/15.
+ */
```

Comment:
I would recommend a small description about the game and a few comments in your code. Overall, your code is well designed. Great job Sarah!